main/guides/zoe/contract-basics.md

```diff
@@ -1,46 +1,47 @@
 # Smart Contract Basics
 
 Before we look at how to make a contract such as the one in [the
-basic dapp](../getting-started/) in the previous section, let's cover some basics.
+basic dapp](../getting-started/) in the previous section, let's cover some basics by writing a simple contract that returns a greetings message. We will simply call it _greetings smart contract_. 
```

Comment:
It's supposed to be a hello world contract. Can we call it that, please?

```suggestion
basic dapp](../getting-started/) in the previous section, let's cover some basics by writing a simple hello-world contract. 
```

Reply:
Thank you for saying that. Fixed.